mainnet-canister-revisions.json

```diff
@@ -80,13 +80,17 @@
     "sha256": "10ced0fec20794640fe45d0e44960faeecaeda8b1b032a04082c758192f24ac6"
   },
   "governance-canister_test": {
-    "rev": "2f87fe95207dc6371a2f2dc273362ba03b41e0e9",
-    "sha256": "1676d35b79dc212f3f52be347f8072d7d2851d59f771da95f3b412f2b39a5439"
+    "rev": "65f66f13fe07b3266ac9d1b44413fd5f0e9b4463",
```

Comment:
This should disappear after a rebase, right?

Reply:
Indeed, gone by now.